test/automated/index.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-env node, mocha, es6 */
+
+'use strict';
+
+const { before, after, describe, it } = require('mocha');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+
+chai.use(chaiAsPromised);
+const { expect } = chai;
+
+require('chromedriver');
+const selenium = require('selenium-webdriver');
+
+describe('Likely', function () {
+    let driver;
+
+    before(function () {
+        driver = new selenium.Builder()
+            .forBrowser('chrome')
+            .build();
+    });
+
+    after(function () {
+        return driver.quit();
+    });
+
+    describe('common operations', function () {
+        before(function () {
+            // The browser could start long
+            this.timeout(20000);
+
+            return driver.get('http://ilyabirman.github.io/Likely/test.html')
+                .then(function () {
+                    // Give Likely time to initialize
+                    return delay(1500);
+                });
+        });
+
+        it('should initialize properly', function () {
+            return expect(driver.findElement({css: '.likely'}).getAttribute('class')).to.eventually.contain('likely_visible likely_ready');
+        });
+
+        it('should fetch the counters for Facebook', function () {
```

Comment:
I think it happens not so often, and it could be broken anyway, by third part, with tests or without.